lib/bin/lwm2m_carrier/lwm2m_carrier.rst

```diff
@@ -4,60 +4,124 @@ LwM2M carrier
 
-Several system resources are used to maintain the device management platform connection.
+   * This is needed for storing keys to the modem, which requires disconnecting and connecting.
+   * The application should not connect the LTE link between a :c:macro:`LWM2M_CARRIER_EVENT_DISCONNECTING` and :c:macro:`LWM2M_CARRIER_EVENT_CONNECTED` event.
 
-While running the module, the application cannot use DTLS through the modem.
 
-Additionally, you must apply workarounds for the following limitations:
+* The Carrier library uses a TLS session for FOTA. TLS handshakes performed by the application might fail if the Carrier library is performing one at the same time.
 
-* TLS through the modem is available, but requires that the application respects the :c:type:`LWM2M_CARRIER_EVENT_FOTA_START` event.
-* TLS handshakes through the modem might fail if the module is performing one at the same time.
-  Implement a retry mechanism so that the application can perform its handshake later.
+   * If the application is in a TLS session and a :c:type:`LWM2M_CARRIER_EVENT_FOTA_START` event is sent to the application, the application must immediately end the TLS session.
+   * TLS will be available again after upon the next :c:type:`LWM2M_CARRIER_EVENT_READY` or :c:type:`LWM2M_CARRIER_EVENT_DEFERRED` event.
+   * The application should Implement a retry mechanism so that the application can perform its handshake later.
+   * Another alternative is to supply TLS from another source than through the modem, for example mbedTLS.
 
-Certification
-*************
-The changelog contains information about the certification status of each version of the library.
-Your final product will typically need certification from the carrier as well.
+Certification and Version Dependencies
+**************************************
+Every released version is considered for certification with carriers.
+
+The LwM2M carrier library is certified together with specific versions of the modem firmware, which is released as part of the |NCS|.
+Refer to the :ref:`liblwm2m_carrier_changelog` to check the certification status of a particular version of the library, and to see what version of |NCS| it was released with.
+The carrier library has an OS abstraction layer. See :file:`lwm2m_os.h`. This makes the Carrier library independent on the underlying implementation.
+
+Your final product will typically need certification from the carrier.
 Contact the carrier for information about their device certification program.
 
+.. _table:
+
++-------------------------+------------------+-------------------------+---------------+
+| Carrier Library Version | Modem Version    | nRF Connect SDK Release | Certification |
+|                         | Dependency       |                         |               |
++=========================+==================+=========================+===============+
+| 0.8.1                   | 1.1.x            | 1.2.0                   | Verizon       |
```

Comment:
Was 0.8.1 released for SDK 1.1.0?

If the target for 0.8.1 was also 1.2.0, maybe we should list both in the cell together?
It is ok to leave out 0.8.1 in favor of 0.8.1+build1. But in that case, it nice to refer to the changelog in addition (to address your other comment) since the table gives a slightly simplified figure for clarity.

Reply:
> Was 0.8.1 released for SDK 1.1.0?

No, it wasn't.
NCS `v1.1.0` contained `liblwm2m_carrier` `v0.8.0`
NCS `v1.2.0` contained `liblwm2m_carrier` `v0.8.1+build1`